Kommunicate/Classes/KMConversationViewConfiguration.swift

```diff
@@ -14,6 +14,8 @@ public struct KMConversationViewConfiguration {
     public var conversationLaunchNotificationName = "ConversationLaunched"
     public var backButtonNotificationName = "ConversationClosed"
     public var isCSATOptionDisabled: Bool = true
+     /// Start new conversation icon in conversation list.
+    public var startNewButtonIcon = UIImage(named: "icon_new_chat_red", in: Bundle.kommunicate, compatibleWith: nil)
```

Comment:
And it will be better to add this icon option in `ALKConfiguration`(ApplozicSwift side). We have hide option in that config, so we should combine these two options and add a new one there.